zookeeper/source/zk_nn_client.cc

```diff
@@ -356,7 +356,12 @@ bool ZkNnClient::add_block(AddBlockRequestProto &req,
   if (znode_data.ecPolicyName == EC_REPLICATION) {
```

Comment:
Basically, there is an issue with storing a std::string field in FileZNode; one of the ReadWrite unit tests mysteriously fails because it cannot free FileZNode for some reason. This issue goes away if this string field is removed from FileZNode. Based on this observation, I pushed a fix that currently comments out all ecPolicyName related logic. Keep your new changes where they are but for now, let's do if (true) {... }. I will make a separate PR to look more into that issue.

Reply:
+1 to this; security team also saw this behavior. Our guess is that `FileZNode` needs to be serializable into a byte sequence for writing into Zookeeper, and there isn't a serialization mechanism for `std::string`s